src/main/java/frc/robot/Subsystems/ClimberWinch.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.Subsystems;
+
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+
+import java.util.function.Supplier;
+
+public class ClimberWinch extends MotoredGenericSubsystem {
+
+    private Supplier<Double> UP_SPEED = rootNamespace.addConstantDouble("up speed", 0.6);
```

Comment:
UP_SPEED, DOWN_SPEED, HOOKED_DOWN_SPEED can all be made public and final, which makes their respective getters redundant

There's a good chance I have no idea what I'm talking about though